game/resource/English/modifier/items/modifier_charge_replenishing.txt

```diff
@@ -0,0 +1,4 @@
+    //=============================================================================
+    // Generic Charge Replenisher
+    //=============================================================================
+            "DOTA_Tooltip_modifier_charge_replenishing"                          "Replenishing charges"
```

Comment:
No, it was supposed to be in my other PR and i fucked it up and then just left it